h/queue.py

```diff
@@ -11,6 +13,8 @@ def __init__(self, namespace, *args, **kwargs):
     def publish(self, topic, data):
         if self.namespace is not None:
             topic = '{0}-{1}'.format(self.namespace, topic)
+        if not isinstance(data, str):
```

Comment:
I'm not sure about this change. Either the `h.queue` module should be responsible for serialisation and deserialisation, or it shouldn't. With this change it's asymmetrical, with the library responsible for serialisation but subscribers responsible for deserialisation.